staging/src/k8s.io/apimachinery/pkg/util/validation/field/errors.go

```diff
@@ -270,3 +270,15 @@ func (list ErrorList) Filter(fns ...utilerrors.Matcher) ErrorList {
 	// FilterOut takes an Aggregate and returns an Aggregate
 	return fromAggregate(err.(utilerrors.Aggregate))
 }
+
+// Is reports whether any Error in err's chain matches target.
+func (v *Error) Is(target error) bool {
```

Comment:
is this part of some interface we want to implement?

Reply:
it's an anonymous interface in the golang `errors` package